src/ansys/dynamicreporting/core/utils/report_remote_server.py

```diff
@@ -37,6 +37,16 @@
 from .encoders import BaseEncoder
 
 
+class TemplateEditorJSONLoadingError(Exception):
```

Comment:
@zhang-yuanrui can you move this into the exception.py file at the same level? That's where the exception classes are stored.

Reply:
Yes, please. And also make it inherit ADRException instead

Reply:
If you do that, you would call the exception with `extra_detail`. You will not need the 'message' parameter if you inherit.

Reply:
ADRException is defined in core directory, while template editor related exceptions are located under utils directory. So I just move this JSON exception to the exceptions.py under the untils directory and have it inherit from RuntimeError.